recipes/transformer-engine/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set version = "1.13" %}
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+{% if cuda_compiler_version in (None, "None", True, False) %}
+{% set cuda_major = 0 %}
+{% else %}
+{% set cuda_major = environ.get("cuda_compiler_version", "11.8").split(".")[0] | int %}
+{% endif %}
+
+package:
+  name: transformer_engine
+  version: {{ version }}
+
+source:
+  - url: https://github.com/NVIDIA/TransformerEngine/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 90dd8f694cba29707c502a4f4d12871cb9d4c1f157d329f22cb3d382de8468ae
+
+  - url: https://github.com/NVIDIA/cudnn-frontend/archive/refs/tags/v1.8.0.tar.gz
+    sha256: 5907bf8baa8f3b74ad430d54027aadd14a0a61166ae4b00c2daece0df384409e
+    folder: 3rdparty/cudnn-frontend
+
+  - url: https://github.com/google/googletest/archive/refs/tags/v1.14.0.tar.gz
+    sha256: 8ad598c73ad796e0d8280b082cebd82a630d73e73cd3c70057938a6501bba5d7
+    folder: 3rdparty/googletest
+
+build:
+  number: 0
+  skip: true  # [win]
+  # pytorch in conda-forge does not support CUDA 11.8 as of v2.5
+  skip: true  # [cuda_compiler_version == "11.8"]
+  skip: true  # [cuda_compiler_version == "None"]
+  script_env:
+    - CUDA_HOME={{ BUILD_PREFIX }}
+    - NVTE_FRAMEWORK=pytorch
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - pytorch                                # [build_platform != target_platform]
+    - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - cuda-nvvm
+    - cmake >=3.21
+    - ninja
+
+  host:
+    - python
+    - pip
+    - pybind11
+    - pytorch
+    - pytorch =*={{ torch_proc_type }}*
+    - setuptools
+    - pydantic
+    - importlib-metadata >=1.0
+    - packaging
+    {% if cuda_major >= 12 %}
+    - cuda-driver-dev
+    - cuda-cudart-dev
+    - cuda-nvrtc-dev
+    - cuda-nvtx-dev
+    - cuda-nvml-dev
+    - libcublas-dev
+    - libcusparse-dev
+    - libcusolver-dev
+    - cudnn
+    - cuda-profiler-api
+    {% endif %}
+
+  run:
+    - python
+    - scipy
+    - pytorch_scatter
```

Comment:
I don't see these as requirements upstream -- should they be here?

Reply:
whoops, probably leftover from another recipe. Removed!